site/source/user-manual/device-guides/limitations/lim-ios.rst

```diff
@@ -10,9 +10,9 @@ iOS
 
 Tor
 ---
-It is not currently possible to run Tor natively (in the background) on iOS. This means that many iOS apps will be unable to connect to your Embassy. The best chance for a Tor background process on iOS in the future is via the `iCepa Project <https://github.com/iCepa>`_.
+Only recently has it become possible to run Tor natively (in the background) on iOS, via :ref:`Orbot<tor-ios>`. Before that, the best chance for a Tor background process on iOS was the `iCepa Project <https://github.com/iCepa>`_.
```

Comment:
I think it's better to just remove this Tor section as it is no longer a 'limitation.'